tools/fake-webserver/server.go

```diff
@@ -88,49 +88,49 @@ type responseOpts struct {
 }
 
 var opts = map[string]map[string]responseOpts{
-	"/api/foo": map[string]responseOpts{
-		"GET": responseOpts{
+	"/api/foo": {
+		"GET": {
```

Comment:
This change doesn't seem covered by the PR or commit description.

Reply:
That's gofumpt simplifying code.